pareto/strategic_water_management/strategic_produced_water_optimization.py

```diff
@@ -5239,30 +5279,40 @@ def water_quality(model):
         doc="Completions Pad Intermediate Flows",
     )
     # Create a set for water quality tracked at the intermediate node between treatment facility and treated water end points
-    treatment_intermediate_label = "-PostTreatmentIntermediateNode"
+    treated_water_label = "-PostTreatmentTreatedWaterNode"
     model.df_sets["TreatedWaterIntermediateNodes"] = [
```

Comment:
I think the renaming you have here makes sense. Should we also rename the set "TreatedWaterIntermediateNodes" to "TreatedWaterNode"?

Reply:
done!